app/models/customer.rb

```diff
@@ -0,0 +1,20 @@
+class Customer < ApplicationRecord
+  has_many :rentals
+
+  validates :id, presence: true
```

Comment:
Requiring `id` to be present means that you can't rely on auto-assignment of IDs and breaks `db:seed`.

This isn't something you'd catch without doing a `rails db:reset` or something though, just something to keep in mind in the future.